geode-core/src/main/java/org/apache/geode/cache/query/internal/RestrictedMethodInvocationAuthorizer.java

```diff
@@ -31,6 +31,11 @@
 
   public static final String UNAUTHORIZED_STRING = "Unauthorized access to method: ";
 
+  protected static final HashMap<String, Set> DEFAULT_WHITELIST;
+  static {
```

Comment:
consider getting rid of the static block. Seems like you could just initialize the static final field with "= createWhiteList();"

Reply:
Thanks. Done.